topsort-catalog-service.yml

```
@@ -10,13 +10,14 @@ info:
   description: Integration for Marketplaces' catalog queries
 
 servers:
-  - url: https://{yourDomain}
+  - url: https://{marketplaceDomain}
```

Maybe something like:
```suggestion
  - url: https://{marketplaceCatalogURL}
```

---

Will leave for another PR